bioblend/galaxy/datasets/__init__.py

```diff
@@ -141,14 +143,81 @@ def download_dataset(self, dataset_id, file_path=None, use_default_filename=True
             # Return location file was saved to
             return file_local_path
 
-    def get_datasets(self, limit=500, offset=0, history_id=None):
+    def get_datasets(
+        self,
+        name: Optional[str] = None,
+        extension: Optional[str] = None,
+        state: Optional[Union[str, List[str]]] = None,
+        deleted: bool = False,
+        visible: bool = True,
+        history_id: Optional[str] = None,
+        workflow_id: Optional[str] = None,
+        job_id: Optional[str] = None,
+        dataset_collection_id: Optional[str] = None,
+        user_id: Optional[str] = None,
+        create_time: Optional[Dict[str, str]] = None,
+        update_time: Optional[Dict[str, str]] = None,
+        file_size: Optional[Dict[str, int]] = None,
```

Comment:
@rikeshi I think you need to update some things here?

Nonetheless it's cool how much is supported by the API already. :+1: